src/sections/chat/live/chat-view.tsx

```diff
@@ -0,0 +1,104 @@
+'use client';
+
+import { useCallback, useRouter, useTranslations } from 'hooks';
+import { Button, Card, Container, Stack, Typography } from '@mui/material';
+import KeyboardArrowLeftIcon from '@mui/icons-material/KeyboardArrowLeft';
+import { Message } from 'types';
+import { CustomLink } from 'components/custom';
+import useTheme from '@mui/system/useTheme';
+import { useSendMessageMutation } from 'store/chat';
+import ChatMessageList from './chat-message-list';
+import ChatMessageInput from './chat-message-input';
+
```

Comment:
imports order

Reply:
And what's wrong with this import order ?

Reply:
on line 3 absolute import before importing the library